client/src/views/SearchResults.tsx

```diff
@@ -0,0 +1,253 @@
+import * as React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import makeStyles from '@mui/styles/makeStyles';
+import * as queryString from 'query-string';
+import Paper from '@mui/material/Paper';
+// import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+import { TextField } from '@mui/material';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+// import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
```

Comment:
remove commented imports

Reply:
these haven't been removed yet

Reply:
![Screenshot 2022-11-04 at 7 25 14 PM](https://user-images.githubusercontent.com/92830192/200097041-cb6a081d-e594-46e8-bda6-c9c6ddba31e6.png)

I was looking at commit [6fa2bbc](https://github.com/Nonprofit-Exchange-Hub/web-app/pull/230/commits/6fa2bbc334d5b86258dfed19363d727991761c32) 8 days ago. Let me know if you see something else and which commit it is. I'll fix it if so.